powerauth-java-crypto/src/main/java/io/getlime/security/powerauth/crypto/lib/encryptor/model/v3/ServerEncryptorSecrets.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * PowerAuth Crypto Library
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getlime.security.powerauth.crypto.lib.encryptor.model.v3;
+
+import io.getlime.security.powerauth.crypto.lib.encryptor.model.EncryptorSecrets;
+import lombok.Data;
+import lombok.Getter;
+
+import java.security.PrivateKey;
+
+/**
+ * The {@code ServerEncryptorSecrets} class provides secret values for server side encryptor using ECIES scheme.
+ * <p>PowerAuth protocol versions:
+ * <ul>
+ *     <li>3.0</li>
+ *     <li>3.1</li>
+ *     <li>3.2</li>
+ * </ul>
+ */
+@Getter
+@Data
+public class ServerEncryptorSecrets implements EncryptorSecrets {
+    /**
+     * Server's private key.
+     */
+    private final PrivateKey serverPrivateKey;
+    /**
+     * Application's secret.
+     */
+    private final String applicationSecret;
```

Comment:
dtto: toString

Reply:
Such secret is not so sensitive, but other properties in the class are. I'll exclude them all.